includes/modules/additional_images.php

```diff
@@ -130,55 +138,40 @@
         $thumb_regular = zen_image($base_image, $products_name, SMALL_IMAGE_WIDTH, SMALL_IMAGE_HEIGHT);
         $large_link = zen_href_link(FILENAME_POPUP_IMAGE_ADDITIONAL, 'pID=' . $_GET['products_id'] . '&pic=' . $i . '&products_image_large_additional=' . $products_image_large);
 
-        // Link Preparation:
-        // -----
-        // This notifier gives notice that an additional image's script link is requested.  A monitoring observer sets
-        // the $p2 value to boolean true if it has provided an alternate form of that link; otherwise, the base code will
-        // create that value.
-        //
-        // $p1 ... (r/o) ... An associative array, containing the 'flag_display_large', 'products_name', 'products_image_large', 'thumb_slashes' and current 'index' values.
-        // $p2 ... (r/w) ... A reference to the $script_link value, set here to boolean false; if an observer modifies that value, the
-        //                     this module's processing is bypassed.
-        // $p3 ... (r/w) ... A reference to the $link_parameters value, which defines the parameters associated with the above
-        //                     link's display.  If the $script_link is updated, these parameters will be used for the display.
-        //
-        $script_link = false;
-        $link_parameters = 'class="additionalImages centeredContent back"' . ' ' . 'style="width:' . $col_width . '%;"';
-        $GLOBALS['zco_notifier']->notify(
```

Comment:
Losing this notification will cause ColorBox and FualSlimBox users to lose that functionality.

Reply:
Oy. I didn't even realize it had been removed in the original PR.

Reply:
Notifier restored.